torch_glow/examples/profiling_executor_example.py

```diff
@@ -0,0 +1,36 @@
+from __future__ import absolute_import, division, print_function, unicode_literals
+
+import torch
+import torch.nn as nn
+import torch_glow
+
+
+class Model(nn.Module):
+    def __init__(self):
+        super(Model, self).__init__()
+        self.linear = nn.Linear(10, 2)
+
+    def forward(self, x):
+        return self.linear(x)
+
+
+torch._C._jit_set_profiling_mode(True)
```

Comment:
Using context manager might be a better example.

Reply:
Yeah, imo it's not worth pulling in jit_utils for it though

Reply:
Oh, yeah. Hmm, we need to move that stuff into `torch.jit`.